### ISSUE TITLE
Ability to switch between available stores in container

### ISSUE THREAD

**Issue by hbrysiewicz** [emberjs/ember-inspector#323]
It would be nice to be able to select different stores within the 'Data' section of the inspector. For example, when using https://github.com/dockyard/ember-admin the models are loaded in a separate admin store. I imagine other scenarios apply as well to when a single app would have multiple stores.

**Comment by teddyzeenny** [comment]
I agree it would be nice to support multiple stores (we can add a dropdown listing the stores in the data tab). I recently had a discussion about this with @tchak and @dgeb.

Here's a workaround to inspect a different store at the moment (for example in ember-admin):

- In the Container tab: Click on `data-adapter` -> `data-adapter:main`. Send it to the console ($E).
- In the console: `$E.set('store', $E.container.lookup('store:admin'))`
- Go back to the inspector, click on the "Data" tab. You are now inspecting `store:admin`

To persist it across reloads, you can do the switch by code: :

```js
// app/routes/application.js
export default Ember.Route.extend({
  beforeModel() {
    this.container.lookup('data-adapter:main').set('store', this.container.lookup('store:admin'));
  }
});
```

These are unfortunately workarounds and a dropdown in the data tab would be great :) The solution will probably be to switch the data adapter's store property through the inspector.
